sep490-frontend/src/app/modules/enterprise/components/building-details/building-details.component.ts

```
@@ -0,0 +1,95 @@
+
+  back(): void {
+    this.location.back();
+  }
+
+  get isEdit(): boolean {
+    return !!this.buildingDetailsStructure.id.value;
+  }
+
+  override reset(): void {
+    if (this.isEdit) {
+      return this.initializeData();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)